frontend/src/redux/selectors.js

```diff
@@ -43,8 +43,6 @@ export const getAge = (state) =>
   new Date().getFullYear() - state.profileReducer.birthYear;
 
 // finances
-export const getSalaryBeforeTax = (state) =>
```

Comment:
if you're going to get rid of it here, you also want to delete the associated values in the state (initialState in the reducer file), in the reducer, and the actions associated with it